### PR TITLE
Extract HTML templates, add zpages.Handler

### PR DESCRIPTION
**Pull request by ramonza**
* Use "esc" to extract HTML templates. This allows them to be edited
  more easily by designers and will facilitate sharing/consistency between
  OpenCensus projects.
* Deprecate installing the z-Pages on http.DefaultServeMux and add a dedicated Handler. This makes links between pages meaningful.
* Added Material Design Lite CSS.

### CODE REVIEW

File: zpages/templates.go [hunk 1]
```diff
@@ -39,127 +47,33 @@ var (
...
+	f, err := statikFS.Open("/templates/" + name + ".html")
+	if err != nil {
+		log.Panicf("%s: %s", name, err.Error())
+	}
+	defer f.Close()
+	text, err := ioutil.ReadAll(f)
+	if err != nil {
+		panic(err)
+	}
+	tmpl, err := template.New(name).Funcs(templateFunctions).Parse(string(text))
```
Comment (odeke-em) [inline zpages/templates.go]:
```go
return template.Must(template.New(name).Funcs(templateFunctions).Parse(string(text)))
```

**Comment by ramonza** [comment]
Thanks @odeke-em, updated

File: zpages/zpages.go [hunk 2]
```diff
@@ -44,9 +44,22 @@ import (
...
 // AddDefaultHTTPHandlers adds handlers for /rpcz and /tracez to the default HTTP request multiplexer.
+// Deprecated: Use Handler.
 func AddDefaultHTTPHandlers() {
 	once.Do(func() {
 		http.HandleFunc("/rpcz", RpczHandler)
 		http.HandleFunc("/tracez", TracezHandler)
 	})
 }
+
+var (
```
Comment (rakyll) [inline zpages/zpages.go]: var Handler http.Handler

File: zpages/templates.go [hunk 3]
```diff
@@ -39,127 +47,29 @@ var (
...
-<th align=right>Min.</th><th align=right>Hr.</th><th align=right>Tot.</th><td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
-<th align=right>Min.</th><th align=right>Hr.</th><th align=right>Tot.</th><td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
-<th align=right>Min.</th><th align=right>Hr.</th><th align=right>Tot.</th><td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
-<th align=right>Min.</th><th align=right>Hr.</th><th align=right>Tot.</th><td>&nbsp;&nbsp;&nbsp;&nbsp;</td>
-<th align=right>Min.</th><th align=right>Hr.</th><th align=right>Tot.</th>
-</tr>
-
-<tr><td colspan=33><font size=-2>&nbsp;</font></td></tr>
+func loadStatik() http.FileSystem {
+	var err error
```
Comment (rakyll) [inline zpages/templates.go]: Remove the line

File: zpages/templates.go [hunk 4]
```diff
@@ -39,127 +47,29 @@ var (
...
-</tr>
-{{end}}
-</table>
-{{end}}
-`
-)
+func parseTemplate(name string) *template.Template {
+	f, err := statikFS.Open("/templates/" + name + ".html")
+	if err != nil {
+		log.Panicf("%s: %s", name, err.Error())
```
Comment (rakyll) [inline zpages/templates.go]: log.Panicf("%v: %v", name, err)

File: zpages/templates.go [hunk 5]
```diff
@@ -39,127 +47,29 @@ var (
...
+	}
+	defer f.Close()
+	text, err := ioutil.ReadAll(f)
+	if err != nil {
+		log.Panicf("%s: %s", name, err.Error())
```
Comment (rakyll) [inline zpages/templates.go]: Same here.

File: zpages/zpages.go [hunk 6]
```diff
@@ -44,9 +44,22 @@ import (
...
+	Handler http.Handler
```
Comment (rakyll) [inline zpages/zpages.go]: Please document what Handler is.

**Comment by ramonza** [comment]
@rakyll replaced statik with esc which embeds the FS impl so doesn't require and external dependency

**Pull request merged**